packages/collector/test/announceCycle/agentHostLookup_test.js

```diff
@@ -210,6 +218,10 @@ describe('agent host lookup state', () => {
       sinonFakeTimers = sinon.useFakeTimers();
     });
 
+    after(() => {
+      sinonFakeTimers.restore();
+    });
```

Comment:
Doing the `restore` here in a `after` hook in this describe at this level makes sure it is executed, even when `RUN_ESM=true`.